### DISCUSSION TITLE
Implement new display management(dde-display) features

### DISCUSSION THREAD

**Discussion by dengbo11** [linuxdeepin/developer-center#3784] [Developer Discussion | 开发者讨论]
SUMMARY | 问题概要
deepin现在的显示管理依赖startdde, x11和wayland显示管理不统一，需要重新实现

SOFTWARE/OS VERSIONS | 软件/系统版本信息
版本无关

PRE-REQUIREMENT | 预置条件
No response

STEPS TO REPRODUCE | 复现步骤
空

OBSERVED RESULT | 观察到的结果
No response

EXPECTED RESULT | 期望的结果
No response

ADDITIONAL INFORMATION | 额外补充
No response

**Comment by BLumia** [comment]
xref: https://github.com/linuxdeepin/developer-center/issues/3785
